### PR TITLE
add PIB dataset

### PR DESCRIPTION
**Pull request by VasudevGupta7**
This pull request will add PIB dataset.

### CODE REVIEW

**Comment by VasudevGupta7** [comment]
Hi, 

I am unable to get success in these tests. Can someone help me by pointing out possible errors?

Thanks

**Comment by lhoestq** [comment]
Hi ! you can read the tests by logging in to circleci.

Anyway for information here are the errors : 
```
datasets/pib/pib.py:19:1: F401 'csv' imported but unused
datasets/pib/pib.py:20:1: F401 'json' imported but unused
datasets/pib/pib.py:36:84: W291 trailing whitespace
```
and 
```
FAILED tests/test_file_encoding.py::TestFileEncoding::test_no_encoding_on_file_open
```

To fix the `test_no_encoding_on_file_open` you just have to specify an encoding while opening a text file. For example `encoding="utf-8"`

File: datasets/pib/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+YAML tags:
+- copy-paste the tags obtained with the tagging app: http://192.168.1.5:8501
```
Comment (lhoestq) [inline datasets/pib/README.md:3]: Could you fill the tags of the dataset card please ?
see https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card for more info

File: datasets/pib/pib.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    def _info(self):
+        # TODO: Specifies the datasets.DatasetInfo object
+        return datasets.DatasetInfo(
+            # This is the description that will appear on the datasets page.
+            description=_DESCRIPTION + f" Available language pairs are {_LanguagePairs}",
+            # This defines the different columns of the dataset and their types
+            features=datasets.Features(
+                {
+                    "src_lang": datasets.Value("string"),
+                    "tgt_lang": datasets.Value("string")
```
Comment (lhoestq) [inline datasets/pib/pib.py:132]: Can you use the Translation feature type instead ?
You can find an example of usage in [flores](https://github.com/huggingface/datasets/blob/master/datasets/flores/flores.py)

**Review by lhoestq — COMMENTED** [review]
Good job thank you !

I just have one comment about the feature type which should be `Translation` :

**Comment by VasudevGupta7** [comment]
Above suggested changes are done.

**Comment by VasudevGupta7** [comment]
All suggested changes are done.

**Comment by lhoestq** [comment]
Nice ! can you re-generate the dataset_infos.json file to take into account the feature type change ?
```
datasets-cli test ./datasets/pib --save_infos --all_configs --ignore_verifications
```
And also format your code ?
```
make style
```

**Review by lhoestq — APPROVED** [review]
Thanks a lot !

**Pull request merged**